### DISCUSSION TITLE
Is Base supporting await (with FastAPI)?

### DISCUSSION THREAD

**Discussion by floatingpurr** [deta/cloud-docs#368] [Help] [answered]
Hi, just wondering if `Base` has support for using `await`. I'm using it in FastAPI but I'm not sure about the behavior of Base in async path operation functions. E.g.:

```python
@app.get('/path')
async def foo():
    # use Base here
```

In the code base there is `AsyncBase` but I've not found any reference about it.

**Comment by jnsougata** [comment, marked as answer]
Ref: https://docs.deta.sh/docs/base/async_sdk

**Reply by floatingpurr** [reply]
Oh, sorry, I hadn't seen it. Thanx!
